wcfsetup/install/files/lib/data/smiley/Smiley.class.php

```diff
@@ -75,26 +77,54 @@ public function getAliases()
     }
 
     /**
-     * Returns the height of the smiley.
+     * Returns the dimensions of the smiley.
      *
-     * @return  int
+     * @since   5.4
+     * @return  int[]
      */
-    public function getHeight()
+    public function getDimensions()
     {
-        if ($this->height === null) {
-            $this->height = 0;
+        if ($this->height === null && $this->width === null) {
```

Comment:
```suggestion
        if ($this->height === null) {
```
This changes makes no sense. If you intended to defend against an unexpected value for `$this->height`, then this should have been a logical "or". Both `$this->height` and `$this->width` equal `null` up until this point, so your extra comparison just makes the code more complex without gaining anything to it.